config/routes.rb

```diff
@@ -6,6 +6,9 @@
   resource :dashboard, only: [:index]
   resource :updates, only: [:create]
 
+  get "/assign_nodes", to: "dashboard#unassigned_nodes"
+  post "/assign_nodes", to: "dashboard#assign_nodes"
```

Comment:
I don't like this routing scheme, since we are putting too much in the dashboard controller and it's not following REST. Since we are in a bit of a hurry, let's go with it, and after the release we can re-evaluate this. Until then, could you add a `TODO` mark ?